### ISSUE TITLE
Add build hook

### ISSUE THREAD

**Issue by jromme** [justinwhall/wp-buildhook-deploy#14]
Hi! 
I can't find a way to add a build hook. Is this possible? Right now it has a production hook. I would like to add a staging build hook.
Thanks in advance!

**Comment by justinwhall** [comment]
Hey @jromme - I'm not sure I quite follow. Are you saying just the _production_ input is visible on the settings page? Could you screenshot that page and post it here?

**Comment by jromme** [comment]
Yes of course I can provide a little more information: I'm actually using the [Gatsby-Toolkit](https://github.com/staticfuse/gatsby-toolkit) but did not find a place where to ask this question (they don't have an issue page). In the Gatsby-Toolkit it shows one input for a build hook. Is there a way to add another input for another build hook?

**Comment by justinwhall** [comment]
Ahh. I see. I turned on issues. Thanks. That plugin only supports _one_ build hook. THis repo, while not maintained, provides two. Didn't seem like it was a feature being used. If it is, we could look at adding a way to add additional build hooks.
